tests/test_set_ocean_surface_temperature.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import os
+import numpy as np
+import fv3gfs.wrapper
+import fv3gfs.util
+from mpi4py import MPI
+from util import (
+    get_default_config,
+    get_state_single_variable,
+    main,
+    replace_state_with_random_values,
+)
+
+
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def mask_non_ocean_values(field):
```

Comment:
Even though I was wrong, I expected `mask_non_ocean_values` to actually select non-ocean values, as opposed to removing them. Could you rename this to `only_ocean_values`, `keep_ocean_values`, or something else without a double-negative?

Reply:
All good -- coupled with your suggestion above, I called this `select_ocean_values`.